include/mbedtls/config_adjust_legacy_crypto.h

```diff
@@ -362,14 +344,6 @@
 #define MBEDTLS_ECP_HAVE_SECP224K1
 #endif
 
-/* Helper symbol to state that the PK module has support for EC keys. This
- * can either be provided through the legacy ECP solution or through the
- * PSA friendly MBEDTLS_PK_USE_PSA_EC_DATA (see pk.h for its description). */
-#if defined(MBEDTLS_ECP_C) || \
-    (defined(MBEDTLS_USE_PSA_CRYPTO) && defined(PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY))
-#define MBEDTLS_PK_HAVE_ECC_KEYS
```

Comment:
MBEDTLS_PK_HAVE_ECC_KEYS still present in framework/scripts/generate_config_tests.py.

Reply:
It's been done in [#34](https://github.com/Mbed-TLS/mbedtls-framework/pull/34) but isn't there the same issue as in [#31](https://github.com/Mbed-TLS/mbedtls-framework/pull/31) about it breaking 3.6? I'm not sure what to do about those two PRs.

Reply:
Ah sorry, I missed that. I know I have to do something about #31. I will have a look to #34 as well.